### PR TITLE
Change mappings structure + new definitions 

### PR DESCRIPTION
**Pull request by brauliorivas**
BEGINRELEASENOTES
- I've modified the `getName`  function from `js/tools`. Instead of using a big switch statement to return the corresponding LaTeX equivalent from the particles id, I've created a new file `data/particles.js` that exports an object with the definitions. The keys are id's converted into strings and the values are the LaTeX equation. This improves readability as the function is much smaller and uses a better structure to store data. Additionally, this doesn't break anything, because when passing a number as key, is coerced to a string so the same.
- I've added more particles to the particle names mappings. I used the [monte carlo scheme](https://pdg.lbl.gov/2007/reviews/montecarlorpp.pdf).
- I also checked the previous mappings, and almost all were correct. I only changed a few. 

ENDRELEASENOTES

### CODE REVIEW

**Comment by tmadlener** [comment]
Thanks a lot for starting on this.

> - I also checked the previous mappings, and almost all were correct. I only changed a few.

Do you have a list of these?

And just for my understanding, did you start from the ones in https://github.com/iLCSoft/LCIO/pull/170 or did you use the ones that are currently in dmx and put that into a table?

**Comment by brauliorivas** [comment]
> Do you have a list of these?

Sorry @tmadlener. I don't have a list. I only remember that I changed **one** name. But don't remember which. 

> did you start from the ones in [iLCSoft/LCIO#170](https://github.com/iLCSoft/LCIO/pull/170) or did you use the ones that are currently in dmx and put that into a table?

I started from the ones currently in dmx and put them into a table.

File: js/tools.js [hunk 1]
```diff
@@ -118,219 +119,10 @@ export function loadMCParticles(jsonData, eventNum,
...
-      return "{\\bar a}_{1}(1260)^{+}";
-    default:
-      console.log("PDG: " + pdg.toString());
-      return "PDG: " + pdg.toString();
-  }
+  const particle = mappings[pdg];
+
+  if (particle !== undefined) return particle;
+
+  console.log("PDG: " + pdg.toString());
```
Comment (tmadlener) [inline js/tools.js:126]: For actual debugging it would be nice, if this also includes the particle name that has been looked up (if any).

**Review by tmadlener — COMMENTED** [review]
> I only remember that I changed one name. But don't remember which.

Alright, then it might just have been a typo in the original implementation.

The ones in iLCSoft/LCIO#170 are using HTML markup directly, so I am not entirely sure how easy it is to directly incorporate them here. @kjvbrt probably has more insights into the possible issues if we combine HTML and Latex here.

**Comment by brauliorivas** [comment]
> The ones in [iLCSoft/LCIO#170](https://github.com/iLCSoft/LCIO/pull/170) are using HTML markup directly, so I am not entirely sure how easy it is to directly incorporate them here.

I can write the LaTeX equivalent without much problem.

File: js/tools.js (hunk 1 above)
Comment (brauliorivas) [reply js/tools.js:126]: So before returning the `particle`, I should add a `console.log` to print `particle` (particle name)?

**Comment by brauliorivas** [comment]
> There will probably be quite some overlap, and there is probably a way to automate it. Don't put too much effort into it yet. With this PR we have the table in place, and we can than in principle decide how we populate it later.

Alright!

File: js/tools.js (hunk 1 above)
Comment (kjvbrt) [reply js/tools.js:129]: console.log() is nice to have for development version (now). For production version it should be removed.

**Comment by kjvbrt** [comment]
Using HTML markup results in shifted sub(sup)scriptions:
image

Instead of:
image

**Pull request merged**